File: 2025/week03/resources/solutions/Question26_solution.py
age = int(input("Veuillez entrer l'age de la personne : "))
if age < 21 :
    print("Entrée : Pas OK")
else :
    if age>25 :
        print("Entrée : OK, Boisson : Pas OK")
    else :
        print("Entrée : OK, Boisson : OK")

File: 2025/week03/resources/solutions/Question26_solution.java
Scanner sc = new Scanner(System.in);
System.out.println("Veuillez entrer l'âge de la personne : ");
int age = sc.nextInt();
if(age<21){
	System.out.println("Entrée : Pas OK");
        }
else{
   	if(age>25){
              	System.out.println("Entrée : OK, Boisson : Pas OK");
            }
       	else{
             	System.out.println("Entrée : OK, Boisson : OK");
            }
        }